hudi-common/src/test/java/org/apache/hudi/common/model/TestPartialUpdateAvroPayload.java

```diff
@@ -55,7 +55,7 @@ public class TestPartialUpdateAvroPayload {
       + "    {\"name\": \"id\", \"type\": [\"null\", \"string\"]},\n"
       + "    {\"name\": \"partition\", \"type\": [\"null\", \"string\"]},\n"
       + "    {\"name\": \"ts\", \"type\": [\"null\", \"long\"]},\n"
-      + "    {\"name\": \"_hoodie_is_deleted\", \"type\": [\"null\", \"boolean\"], \"default\":false},\n"
+      + "    {\"name\": \"_hoodie_is_deleted\", \"type\": [\"boolean\", \"null\"], \"default\":false},\n"
```

Comment:
ditto

Reply:
actually `_hoodie_is_deleted` should never be null. let's make it just boolean type?

Reply:
> actually `_hoodie_is_deleted` should never be null. let's make it just boolean type?

done